data/core.yaml

```diff
@@ -1973,16 +1973,16 @@ en:
       search_cafe: "There are many different features that can be represented by points. The point you just added is a cafe. **Search for '{preset}'.**"
       choose_cafe: "**Choose {preset} from the list.**"
       feature_editor: "The point is now marked as a cafe. Using the feature editor, we can add more information about the cafe."
-      fields_info: "In OpenStreetMap, all of the fields are optional, and it's OK to leave a field blank if you are unsure."
+      fields_info: "In OpenHistoricalMap, all of the fields are optional, and it's OK to leave a field blank if you are unsure."
```

Comment:
Should we be encouraging mappers to leave the Start Date and End Date fields blank if they’re unsure about the exact date? In my own mapping, I’ve been making sure to always fill them in with something, to keep them from showing up throughout time in the vector tiles, but using the `start_date:edtf` and `end_date:edtf` fields to indicate any uncertainty. We can’t recommend the EDTF keys to novice mappers just yet, but maybe we could recommend describing any uncertainty in the Note field.